packages/api/src/external/commonwell/admin/patch-patient-duplicates.ts

```diff
@@ -0,0 +1,74 @@
+import { getPersonId, getPersonIdFromUrl } from "@metriport/commonwell-sdk";
+import { getPatientOrFail } from "../../../command/medical/patient/get-patient";
+import MetriportError from "../../../errors/metriport-error";
+import { Util } from "../../../shared/util";
+import { setCommonwellId } from "../patient-external-data";
+import { getCWAccessForPatient } from "./shared";
+
+/**
+ * IMPORTANT: This is an Admin-only function, it should not be executed outside of internal endpoints.
+ *
+ * Patch duplicated persons in CommonWell - sibling of find-patient-duplicates.ts
+ *
+ * Links the patient to the chosen person.
+ * Additionally, unenroll:
+ * - any other person linked to the patient; AND
```

Comment:
Shouldn't we only be unenrolling persons that we enroll? AFAIK when we raise cases CW makes the implementer unenroll the dups that they create.

Reply:
I was planning to only dedup the ones with persons we create. But now that you mention this I think its a good idea, since this change will allow us to point our patients to the right person even though the dup was not created by us - although I'm not sure how often this happens (someone else creates a dup and our patient is linked to that person).

When thinking about this change I was concerned with the `enroller` name, if its consistently `Metriport` across envs, but looking at integration's orgs I see we're also `Metriport` there, so I'll make this change.

Reply:
Actually, the `enroller` is our cx's name, and we can check it easily since have that info to build the auth w/ CW.